release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -495,6 +495,18 @@ The egress router CNI plug-in is generally available. The Cluster Network Operat
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/using-an-egress-router-ovn.adoc[Considerations for the use of an egress router pod].
 
+[id="ocp-4-8-ingress-controller-power-of-two-random-choices"]
+==== Power of Two Random Choices balancing algorithm for Ingress Controller
+
+{product-title} Ingress Controllers now uses the link:https://www.haproxy.com/blog/power-of-two-load-balancing/[Power of Two Random Choices] balancing algorithm by default for more even balancing after reloading. As in earlier releases, the balancing algorithm for a specific route can still be configured using the `haproxy.router.openshift.io/balance` route annotation. See xref:../networking/routes/route-configuration.adoc#nw-route-specific-annotations_route-configuration[Route-specific annotations] for more information.
```

Comment:
"Ingress Controllers now uses": subj-obj seems off here unless it's due to product name weirdness